pipelines/tekton/build-container-image-pipeline/build-container-image-pipeline.yaml

```diff
@@ -19,7 +19,37 @@ spec:
     type: string
   - name: containerfileRelativePath
     type: string
+  - name: modelRelativePath
+    type: string
+  - name: fetch-model
+    type: string
+  - name: git-model-repo
+    type: string
+    default: ""
+  - name: git-revision
+    type: string
+    default: "main"
   tasks:
+  - name: git-clone-model-repo
+    taskRef:
+      kind: ClusterTask
+      name: git-clone
+    params:
+    - name: url
+      value: $(params.git-model-repo)
+    - name: revision
+      value: $(params.git-revision)
+    - name: gitInitImage
+      value: registry.redhat.io/openshift-pipelines/pipelines-git-init-rhel8@sha256:1a50511583fc02a27012d17d942e247813404104ddd282d7e26f99765174392c
+    - name: subdirectory
+      value: /model_dir/
```

Comment:
Something about the static `model_dir` path is  wrong: https://github.com/opendatahub-io/ai-edge/issues/175.